incubator/vault/values.yaml

```diff
@@ -0,0 +1,61 @@
+# Default values for vault.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+replicaCount: 1
+image:
+  repository: vault
+  tag: 0.8.1
+  pullPolicy: IfNotPresent
+service:
+  name: vault
+  type: ClusterIP
+  port: 8200
+ingress:
+  enabled: false
+  # Used to create Ingress record (should used with service.type: ClusterIP).
+  hosts:
+    - chart-example.local
+  annotations:
+    # kubernetes.io/ingress.class: nginx
+    # kubernetes.io/tls-acme: "true"
+  tls:
+    # Secrets must be manually created in the namespace.
+    # - secretName: chart-example-tls
+    #   hosts:
+    #     - chart-example.local
+# A YAML representation of a final vault config.json file.
+# See https://www.vaultproject.io/docs/configuration/ for more information.
+vaultmode:
+# Only used to enable dev mode. When in dev mode, the rest of this configuration
```

Comment:
What do you mean by "rest of this configuration is ignored"? Are you referring to all the options under `vaultmode`? Also, I think it might read a little better if you had:

```yaml
vault:
    dev: true
    config:
        listener:
            ...
        storage:
            ...
```

That gives a more of a namespace to add other vault options that are outside the config file.